src/PreviewWebPanel.ts

```diff
@@ -124,10 +145,12 @@ function getWebviewContent(context: vscode.ExtensionContext, webview: vscode.Web
       }
       </style>
     </head>
+
     <body x-timestamp="${Date.now()}">
-      
       <div id="asyncapi"></div>
-  
+      <pre class="mermaid">
```

Comment:
I guess this would be on a different preview-panel, started by a different command..
Good if it helps you start seeing something..